### PR TITLE
Add some guidance to prevent parallel encryption

### PR DESCRIPTION
**Pull request by wussler**
Addresses #2

### CODE REVIEW

File: draft-wussler-openpgp-pqc.md [hunk 1]
```diff
@@ -564,7 +564,15 @@ As explained in {{non-composite-multi-alg}}, the OpenPGP protocol inherently
 supports parallel encryption to different keys of the same recipient.
 Implementations MUST NOT encrypt a message to a purely traditional public-key
 encryption key of a recipient if it is encrypted to a PQ/T key of the same
-recipient.
+recipient. This can be achieved with checks at different levels, for instance:
+
+- By ensuring only one key per recipient is used.
+
+- By checking if there is a "Superseded by" signature that links to another
```
Comment (falko-strenzke) [inline draft-wussler-openpgp-pqc.md:571]: This seems to refer to this proposal: https://gitlab.com/openpgp-wg/rfc4880bis/-/issues/64

But this is not yet part of the crypto-refresh. So I would prefer not to include it here until it becomes adapted into the cr-re

File: draft-wussler-openpgp-pqc.md [hunk 2]
```diff
@@ -564,7 +564,15 @@ As explained in {{non-composite-multi-alg}}, the OpenPGP protocol inherently
 supports parallel encryption to different keys of the same recipient.
 Implementations MUST NOT encrypt a message to a purely traditional public-key
 encryption key of a recipient if it is encrypted to a PQ/T key of the same
-recipient.
+recipient. This can be achieved with checks at different levels, for instance:
+
+- By ensuring only one key per recipient is used.
```
Comment (falko-strenzke) [inline draft-wussler-openpgp-pqc.md:569]: How is this check to performed exactly? By comparing the user IDs of the keys?

**Comment by wussler** [comment]
Seems like [!222](https://gitlab.com/openpgp-wg/rfc4880bis/-/merge_requests/222) is not getting into the C-R. I would therefore propose to add it to this document and in this PR 

WDYT @falko-strenzke, @fluppe2, and @TJ-91?

**Comment by fluppe2** [comment]
The [!222](https://gitlab.com/openpgp-wg/rfc4880bis/-/merge_requests/222) feels more like a general feature request and I'm kind of opposed to blur the line between general stuff and the pqc extension that we are defining.

**Comment by wussler** [comment]
Sure, but then would it have to be a standalone RFC?

**Comment by fluppe2** [comment]
I dunno, in my personal opinion that would depend on the general future structure of OpenPGP if it stays a monolithic block or some potential features can be and are discussed as extensions.

**Comment by falko-strenzke** [comment]
I would also favour to restrict our draft to the PQC-specific features and thus not to address the extension from [!222](https://gitlab.com/openpgp-wg/rfc4880bis/-/merge_requests/222).

**Comment by wussler** [comment]
Good for me, I'll propose this in the rechartering as a standalone topic looking for a friend.

**Review by fluppe2 — COMMENTED** [review]
Shall we close this PR then?

**Comment by wussler** [comment]
This PR is supposed to be discussed after adoption, to understand what kind of guidance (if any) is due here

**Pull request closed without merging**